e2e-tests/search-test.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test, expect } from '@playwright/test';
+
+test('Test fetching Kristiansund - Molde and loading more after first result', async ({
+  page,
+}) => {
+  await page.goto(process.env.E2E_URL ?? 'http://localhost:3000');
+  await page.getByRole('textbox', { name: 'From' }).click();
+  await page.getByRole('textbox', { name: 'From' }).fill('Kristiansund');
+  await page.getByRole('option', { name: 'Kristiansund Kristiansund' }).click();
+  await page.getByRole('textbox', { name: 'To' }).click();
+  await page.getByRole('textbox', { name: 'To' }).click();
```

Comment:
Is there a reason for why the exact same line of code is repeated twice on line 10 and line 11?🤔

Reply:
Probably hiccup with the recorder. Can remove